src/Components/CreateImageWizard/formComponents/Oscap.js

```diff
@@ -73,11 +81,22 @@ const ProfileSelector = ({ input, showSelector }) => {
   };
 
   return (
-    <FormGroup
-      isRequired={true}
-      label={'Profile to use for this image'}
-      data-testid="profiles-form-group"
-    >
+    <FormGroup isRequired={true} data-testid="profiles-form-group">
+      {'OpenSCAP profile'}
+      <Popover
+        maxWidth="30rem"
+        position="left"
+        bodyContent={
+          <TextContent>
+            <Text>
+              Required selection for OpenSCAP, in order to make this work, user
+              cannot clear this field, the default value is None.
+            </Text>
+          </TextContent>
+        }
+      >
+        <HelpIcon className="pf-u-ml-sm" />
```

Comment:
We should always avoid adding css whenever possible... which is actually most of the time, fortunately. :smile: 

```
<Button variant="plain" aria-label="About OpenSCAP">
    <HelpIcon />
</Button>
```